velox/exec/tests/CMakeLists.txt

```diff
@@ -99,6 +99,8 @@ add_test(
   COMMAND velox_exec_test
   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 
+set_tests_properties(velox_exec_test PROPERTIES TIMEOUT 3000)
```

Comment:
What units is it? Seconds?
So 50 minutes?

Reply:
Yes, the build went through and I saw the timeout correctly set now

test 278
        Start 278: velox_exec_test

278: Test command: /root/project/_build/debug/velox/exec/tests/velox_exec_test
278: Test timeout computed to be: 3000